modules/dxp/apps/portal-workflow/portal-workflow-metrics-api/src/main/java/com/liferay/portal/workflow/metrics/util/comparator/WorkflowMetricsSLADefinitionVersionIdComparator.java

```diff
@@ -14,12 +14,26 @@
 public class WorkflowMetricsSLADefinitionVersionIdComparator
 	extends OrderByComparator<WorkflowMetricsSLADefinitionVersion> {
 
-	public WorkflowMetricsSLADefinitionVersionIdComparator() {
-		this(false);
-	}
+	public static final String ORDER_BY_ASC =
```

Comment:
@Valmir-unicap Why making this change? I think we are going to keep these as is?

Reply:
Sorry, erro my! I'll fix it now!